ts/model/ElasticSearchQuery.ts

```diff
@@ -0,0 +1,156 @@
+/* tslint:disable:max-classes-per-file */
```

Comment:
Is all of this really necessary? Seems very heavy for just a set of sample query commands.